slo_generator/compute.py

```diff
@@ -28,12 +29,12 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def compute(slo_config,
-            config,
-            timestamp=None,
+def compute(slo_config: dict,
+            config: dict,
+            timestamp: Optional[float] = None,
```

Comment:
Not sure we need the `Optional` there. The default value already indicates that the value is optional. From my experience, we mostly use `Optional` on return types but I have done an extensive research on this topic. Are you aware of any best practice or article we could use as a reference?

Reply:
Yeah you right, I got some errors about this in first time I thought it's required, but I think it's not 😅 I have fixed this

Reply:
https://docs.python.org/3/library/typing.html#typing.Optional says it is OK to use `Optional` when `None` is a valid value.